content/azure_chef_cli.md

```diff
@@ -208,6 +208,10 @@ The extension has the following options that can be provided in the
 :   Specifies the [Ohai Hints](/ohai/#hints) to be set in the Ohai configuration of
     the target node.
 
+`chef_package_url`
+
+:   Specifies a url to download Chef Infra Client package (.msi .rpm .deb) and
+    subsequently install.
```

Comment:
```suggestion
:   Specifies a URL to download and install the Chef Infra Client package (.msi .rpm .deb) from.
```

Reply:
Thank you @IanMadd. Updated as per review comments and done rebase.